### PR TITLE
Install Twitter share button

### LINKED ISSUE #13: Make it easier to share
By placing the Twitter Share link you can create situations that are easy to share. 

Let's be able to share images and URL!

### PR DESCRIPTION
**Pull request by konojunya**
Using Twitter API, make HTML canvas image data and make it easy to share.

close #13

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -151,14 +156,23 @@ class App extends Component {
...
           >
             Click here
           </button>
+          or
+          <button
+            className="App-twitter-button"
+            onClick={this.shareToTwitter}
+            type="button"
+          >
+            Share to Twitter
```
Comment (catalinmiron) [inline src/App.js]: I think this should be `Share on Twitter`

File: src/App.js [hunk 2]
```diff
@@ -64,6 +64,11 @@ class App extends Component {
     download(this.canvas);
   };
 
+  shareToTwitter = e => {
+    e.preventDefault();
+    shareToTwitter(this.canvas);
```
Comment (catalinmiron) [inline src/App.js]: the naming is confusing :) I think that naming callback should start with `on` prefix, like for example `onShareTwitter`. What do you think?

File: src/App.js (hunk 1 above)
Comment (catalinmiron) [reply src/App.js]: :) no problem. We’re here to help each other

File: src/App.js (hunk 2 above)
Comment (konojunya) [reply src/App.js]: I think so, too :)

File: src/App.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import { drawContributions } from "github-contributions-canvas";
-import { download } from "./utils/export";
+import { download, onShareTwitter } from "./utils/export";
```
Comment (catalinmiron) [inline src/App.js]: I'm not sure if you should change the name of this method :) only callbacks inside the component should begin with `on` prefix. From my point of view, `postMediaTweet` or `postTweet` should be a better name.

Naming is really hard :)

Comment (sallar) [reply src/App.js]: I agree. @konojunya let's change the name to "`uploadToTwitter`"

Comment (konojunya) [reply src/App.js]: ok!
Thank you for careful code review :)

**Comment by sallar** [comment]
@konojunya @catalinmiron Im merging this into another branch to do some cleanup. Good work guys!

**Pull request merged**